models/core/src/core.ts

```diff
@@ -173,6 +173,10 @@ export class TType extends TObj implements Type<any> {
 @Model(core.class.TypeString, core.class.Type)
 export class TTypeString extends TType {}
 
+@UX(core.string.String)
+@Model(core.class.TypeRecord, core.class.Type)
+export class TTypeRecord extends TType {}
```

Comment:
Why we need it?

Reply:
It was and it throw error because it is missing. Probable need to remove it.